src/run.ts

```diff
@@ -36,13 +36,52 @@ type Args = {
   quiet: boolean;
   disableTelemetry: boolean;
   promoCode?: string;
-
+  preSelectedProject?: {
+    authToken: string;
+    selfHosted: boolean;
+    dsn: string;
+    projectId: string;
+    projectSlug: string;
+    projectName: string;
+    orgId: string;
+    orgName: string;
+    orgSlug: string;
+  };
```

Comment:
Note: This overlaps with https://github.com/getsentry/sentry-wizard/pull/671. So if `preSelectedProject` is provided, the newly added `--orgSlug` and `--projectSlug` will not have any effect on the wizard behaviour.